src/libs/actions/IOU.ts

```diff
@@ -5393,7 +5400,203 @@ function deleteMoneyRequest(transactionID: string, reportAction: OnyxTypes.Repor
         updatedReportPreviewAction.childMoneyRequestCount = reportPreviewAction.childMoneyRequestCount - 1;
     }
 
-    // STEP 5: Build Onyx data
+    let urlToNavigateBack: ReturnType<typeof ROUTES.REPORT_WITH_ID.getRoute> | undefined;
+
+    // STEP 5: Calculate what is the url that we will navigate user back
```

Comment:
Re-word slightly for better readability `// STEP 5: Calculate the url that the user will be navigated back to`